lib/updateDataDisplay.ml

```diff
@@ -48,7 +56,7 @@ let print_mod_ok conf base =
     match List.assoc_opt "max_nb_update" conf.base_env with
     | Some n ->
         let n = int_of_string n in
-        if n > 50000 then 5000 else n
+        if n > 5000 then 5000 else n
     | _ -> 5000
```

Comment:
what is max_nb_update?
maybe make a gloabal variable `let default_max_nb_update = 5000"